tinygrad/tensor.py

```diff
@@ -105,10 +105,11 @@ class Tensor:
   training: ClassVar[bool] = False
   no_grad: ClassVar[bool] = False
 
-  def __init__(self, data:Union[None, ConstType, List, Tuple, LazyBuffer, np.ndarray, bytes, MultiLazyBuffer, Variable],
+  def __init__(self, data:Union[None, ConstType, List, Tuple, LazyBuffer, np.ndarray, bytes, MultiLazyBuffer, Variable, pathlib.Path],
```

Comment:
You need to test this functionality too.

Reply:
I added some tests for the path, also changed how dtype parameter is handled as with the previous implementation only `uint8` was supported